comp/core/secrets/secretsimpl/secrets_test.go

```diff
@@ -85,14 +63,14 @@ instances:
 	testConfOrigin = handleToContext{
 		"pass1": []secretContext{
 			{
-				origin:   "test",
-				yamlPath: "instances/password",
+				origin: "test",
+				path:   []string{"instances", "0", "password"},
```

Comment:
The `0` and `1` changes is because of how yaml handle nested values?

Reply:
Yes, these are indexes into an array of values. Before, these indexes would just be ignored, but now they're part of the path, because the new method `configAssignAtPath` is able to make use of them to modify a single element of an array.